From fb033fc4ad3e552abb6cb4b6e8b4510a443790c9 Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Wed, 11 Jul 2018 13:18:29 +0300
Subject: [PATCH] board: habana: move apt update before packages

'apt update' should come before installing any packages.
Move 'apt update' to the top of the function.

Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/board/habanalabs/common/local_install_packages b/board/habanalabs/common/local_install_packages
index 3a0fad5..4e44f79 100755
--- a/board/habanalabs/common/local_install_packages
+++ b/board/habanalabs/common/local_install_packages
@@ -38,9 +38,9 @@
 
 function local_install_packages() {
 	local_msg "************ Installing habana prerequisites ***********"
+	sudo apt update				|| return 1;
 	sudo apt-get install ${pre_pkg_lst} -y	|| return 1;
 	local_msg "************ Installing new habana packages ************"
-	sudo apt update				|| return 1;
 	sudo apt install ${habana_pkg_lst} -y	|| return 1;
 }
 
